setup.py

```diff
@@ -1,11 +1,23 @@
 from setuptools import setup, find_packages
 from setuptools.command.test import test as TestCommand
 import sys
-
-if sys.version_info.major != 3:
-    print('BiblioPixel v3 is no longer supported on Python 2!')
-    print('If you need Python2 support please insteall BiblioPixel v2')
-    print('pip install "bibliopixel<3.0"')
+from sys import version_info as vi
+
+if vi.major != 3:
+    ver = '{}.{}.{}'.format(vi.major, vi.minor, vi.micro)
```

Comment:
Kept this in for historical reasons but this comment is now subsumed in the final sum up....